src/frontend/src/views/Data/components/LineageTooltip/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { Label, Text, Wrapper } from "./style";
+
+export interface Lineage {
+  last_updated: string;
+  lineage: string;
+  probability: string;
+  version: string;
+}
+
+interface Props {
+  lineage: Lineage;
+}
+
+const DISPLAY_ORDER: Array<keyof Lineage> = [
+  "lineage",
+  "probability",
+  "version",
+  "last_updated",
+];
+
+export const LineageTooltip = ({ lineage }: Props): JSX.Element => {
```

Comment:
This component renders the details of the Lineage tooltip and in the expected row order and label texts
<img width="459" alt="Screen Shot 2021-05-10 at 1 02 11 PM" src="https://user-images.githubusercontent.com/6309723/117721149-59838600-b194-11eb-8d94-6f0567f4893d.png">